packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1622,6 +1623,7 @@ export class MergeTree {
 		}
 
 		if (
+			// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- ?? is not logically equivalent when the first clause returns false.
```

Comment:
I almost wonder if we should disable this rule, given how frequently it makes invalid recommendations. I've been burned by its auto-fix behavior in particular before.

Reply:
I agree. Letting the linter fix things should be almost a no-brainer, but this particular "fix" requires more careful consideration on a case-by-case basis.